### ISSUE TITLE
SIGSEGV: segmentation violation

### ISSUE THREAD

**Issue by ncmans** [pomerium/pomerium#916]
## What happened?

Panic at runtime

## What did you expect to happen?

No panic

## How'd it happen?

Without any particular external action, just after a while of running.

## What's your environment like?

- Pomerium version (retrieve with `pomerium --version` or `/ping` endpoint): 0.8.0
- Server Operating System/Architecture/Cloud: Kubernetes GKE
- Helm Version: 9.0.0

```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.0", GitCommit:"2bd9643cee5b3b3a5ecbd3af49d09018f0773c77", GitTreeState:"clean", BuildDate:"2019-09-18T14:36:53Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"15+", GitVersion:"v1.15.11-gke.5", GitCommit:"a5bf731ea129336a3cf32c3375317b3a626919d7", GitTreeState:"clean", BuildDate:"2020-03-31T02:49:49Z", GoVersion:"go1.12.17b4", Compiler:"gc", Platform:"linux/amd64"}
```

## What's your config.yaml?

```config.yaml
metrics_address: :9090
forward_auth_url: https://forwardauth.abc.xyz
policy:
  - allowed_domains:
    - abc.xyz
    from: https://alertmanager.abc.xyz
    to: http://prometheus-alertmanager.infra:80
  - allowed_domains:
    - abc.xyz
    from: https://cockroachdb.abc.xyz
    to: http://crdb-cockroachdb.infra:8080
  - allowed_domains:
    - abc.xyz
    from: https://grafana.abc.xyz
    to: http://grafana.infra:80
  - allowed_domains:
    - abc.xyz
    from: https://prometheus.abc.xyz
    to: http://prometheus-server.infra:80
cookie_secret: abcde
shared_secret: abcde
idp_client_id: abcde
idp_client_secret: abcde
```

```
# container spec
  containers:
  - args:
    - --config=/etc/pomerium/config.yaml
    env:
    - name: SERVICES
      value: authorize
    - name: CERTIFICATE_FILE
      value: /pomerium/cert.pem
    - name: CERTIFICATE_KEY_FILE
      value: /pomerium/privkey.pem
    - name: CERTIFICATE_AUTHORITY_FILE
      value: /pomerium/ca.pem
    image: pomerium/pomerium:v0.8.0
```

## What did you see in the logs?

```logs
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xee8339]

goroutine 18037 [running]:
github.com/pomerium/pomerium/authorize.(*Authorize).IsAuthorized(0xc0004bb320, 0x13fa740, 0xc000625e60, 0xc000636750, 0x0, 0x0, 0x0)
	/home/runner/work/pomerium/pomerium/authorize/grpc.go:35 +0x3e9
github.com/pomerium/pomerium/internal/grpc/authorize._Authorizer_IsAuthorized_Handler(0x10a5400, 0xc0004bb320, 0x13fa740, 0xc000625e60, 0xc0008c5200, 0x0, 0x13fa740, 0xc000625e60, 0xc0001f8c00, 0xb45)
	/home/runner/work/pomerium/pomerium/internal/grpc/authorize/authorize.pb.go:349 +0x217
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0000a5d40, 0x140ab80, 0xc00084e600, 0xc000522700, 0xc0002c0e40, 0x1c72bb0, 0x0, 0x0, 0x0)
	/home/runner/go/pkg/mod/google.golang.org/grpc@v1.28.0/server.go:1082 +0x50a
google.golang.org/grpc.(*Server).handleStream(0xc0000a5d40, 0x140ab80, 0xc00084e600, 0xc000522700, 0x0)
	/home/runner/go/pkg/mod/google.golang.org/grpc@v1.28.0/server.go:1405 +0xcc9
google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc0009c4510, 0xc0000a5d40, 0x140ab80, 0xc00084e600, 0xc000522700)
	/home/runner/go/pkg/mod/google.golang.org/grpc@v1.28.0/server.go:746 +0xa1
created by google.golang.org/grpc.(*Server).serveStreams.func1
	/home/runner/go/pkg/mod/google.golang.org/grpc@v1.28.0/server.go:744 +0xa1
```

**Comment by ncmans** [comment]
> This is duplicated of. #704
> 
> @ncmans this is fixed in v0.8.1

Thanks — will test and report back

**Comment by desimone** [comment]
Closing for now. @ncmans let us know if that doesn't do the trick
